presto/tests/test_presto.py

```diff
@@ -1,11 +1,32 @@
 # (C) Datadog, Inc. 2019-present
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+import prestodb
 import pytest
 
+from datadog_checks.dev.jmx import JVM_E2E_METRICS
+from datadog_checks.dev.utils import get_metadata_metrics
 
-# We don't test JMXFetch based integrations in this repo
-# This is required to allow E2E to spin up the environment
-@pytest.mark.usefixtures('dd_environment')
-def test():
-    pass
+from .common import METRICS
+
+
+def make_query():
```

Comment:
We can move make_query to `conftest.py` and add it as a condition in docker_run.
https://github.com/DataDog/integrations-core/blob/cf9b128fea06c355aa7d30043cc75088b31535b6/datadog_checks_dev/datadog_checks/dev/docker.py#L141

Reply:
Sure!